.github/workflows/generate-dsl-info.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate DSL Info
+
+permissions:
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  generate-dsl-info:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Generate DSL info
+        run: npm run generate-dsl-info
```

Comment:
I think we can merge these generating actions like do `pnpm run generate:*` here

Reply:
Do i need any additional settings using `pnpm run generate:*` ?

i got an error `no matches found: generate:*` so i add a `pnpm run generate:all` script.

Reply:
Ah sorry. i think in pnpm, it's `pnpm run "/^generate:.*/" --parallel`

Reply:
also you might've forgot to commit the deletions of old workflow files

Reply:
ok that works! fixed now.